src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -126,6 +130,27 @@ public Response createDataAccessRequest(
     }
   }
 
+  @VisibleForTesting
+  public void validateInternalCollaborators(DataAccessRequest payload, User requestingUser) {
+    Integer institution = requestingUser.getInstitutionId();
+    List<Collaborator> internalCollaborators = payload.getData().getInternalCollaborators();
+    if (internalCollaborators != null && !internalCollaborators.isEmpty()) {
+      for (Collaborator collaborator : internalCollaborators) {
```

Comment:
Looking at `DataAccessRequestData.getInternalCollaborators()`, `internalCollaborators` will never be `null`. Also there's no need to check for an empty list since the `for` loop won't be executed if the list is empty.
```suggestion
    for (Collaborator collaborator : internalCollaborators) {
```